docs/developers/deployment_guides/blockchain/oracles.md

```diff
@@ -125,7 +131,8 @@ cd ../sp-oracle/script/
 docker compose up build
 ```
 
-A `run` folder will be created with artifacts for the future solution.
+A `run` folder will be created with artifacts for the future solution. It should consist of non-empty folders: `dist`, `node_modules`.
```

Comment:
A `run` folder will be created, containing folders `dist` and `node_modules` with the artifacts for the future solution.